flight/Modules/Actuator/actuator.c

```diff
@@ -340,12 +340,23 @@ static void actuatorTask(void* parameters)
 				continue;
 			}
 
-			if((mixers[ct].type == MIXERSETTINGS_MIXER1TYPE_MOTOR) || (mixers[ct].type == MIXERSETTINGS_MIXER1TYPE_SERVO))
+			if((mixers[ct].type == MIXERSETTINGS_MIXER1TYPE_MOTOR) || (mixers[ct].type == MIXERSETTINGS_MIXER1TYPE_SERVO)) {
 				status[ct] = ProcessMixer(ct, curve1, curve2, &mixerSettings, &desired, dT);
-			else
-				status[ct] = -1;
-
 
+				// If a motor type, additional work to be done
+				if(mixers[ct].type == MIXERSETTINGS_MIXER1TYPE_MOTOR) {
+					if (status[ct] > 0) {
+						// Apply curve fitting, mapping the input to the propeller output.
+						status[ct] = actuatorSettings.MotorInputOutputCurveFit[ACTUATORSETTINGS_MOTORINPUTOUTPUTCURVEFIT_A] *
```

Comment:
Generally, we've had a strong principle about not adding code that doesn't provide complete functionality. But handling the negative case is logical so I can go either way, and am fine with whatever the reviewer(s) decide(s).

> prop may not respond exactly the same in reverse depending on geometry of things around it and whether it's a symmetric foil, but it ought to be close

I expect it to be **very**, **very** different. We've never formally tested our props running backwards, but theoretically there should be a tremendous difference on non-symetrical props. 

Assuming my above assertion is correct, one interesting thing of note is that we will have ```A_reversed << 1``` in the case of reverse props, assuming that ```status[idx]=1``` represents maximum forward thrust, and thus ```status[idx]=-1``` should represent an equal number of Newtons of reverse thrust.

(Not sure if the above sentence is clear, if it's not I'll take another stab at explaining it.)